src/timeseriesflattener/flattened_dataset.py

```diff
@@ -68,13 +86,14 @@ def _override_cache_attributes_with_self_attributes(
     def __init__(  # pylint: disable=too-many-arguments
         self,
         prediction_times_df: DataFrame,
+        drop_pred_times_with_insufficient_look_distance: bool,  # noqa
         cache: Optional[FeatureCache] = None,
         id_col_name: str = "dw_ek_borger",
```

Comment:
this default should be removed everywhere (also in specs) -- probably a different PR though